src/og/control/ruler/RulerScene.js

```diff
@@ -112,12 +112,14 @@ class RulerScene extends RenderNode {
             entities: [this._trackEntity, this._propsLabel],
             pickingEnabled: false,
             polygonOffsetUnits: -1.0,
-            relativeToGround: true
+            relativeToGround: true,
+            displayInLayerSwitcher: false
```

Comment:
Nice!

Reply:
Also `displayInLayerSwitcher` must be in the `Layer `class constructor.

Reply:
I think it is already. I added it to the  @params of Layer.js